src/psyclone/transformations.py

```diff
@@ -313,12 +313,161 @@ def apply(self, node, options=None):
         return schedule, keep
 
 
+class OMPTaskloopTrans(ParallelLoopTrans):
+
+    '''
+    Adds an OpenMP taskloop directive to a loop.
+
+    TODO: #1364 Taskloops do not yet support reduction clauses.
+
+    For example:
+
+    >>> from pysclone.parse.algorithm import parse
+    >>> from psyclone.psyGen import PSyFactory
+    >>> api = "gocean1.0"
+    >>> filename = "nemolite2d_alg.f90"
+    >>> ast, invokeInfo = parse(filename, api=api, invoke_name="invoke")
+    >>> psy = PSyFactory(api).create(invokeInfo)
+    >>>
+    >>> from psyclone.transformations import OMPParallelTrans, OMPSingleTrans
+    >>> from psyclone.transformations import OMPTaskloopTrans
+    >>> singletrans = OMPSingleTrans()
+    >>> paralleltrans = OMPParallelTrans()
+    >>> tasklooptrans = OMPTaskloopTrans()
+    >>>
+    >>> schedule = psy.invokes.get('invoke_0').schedule
+    >>> schedule.view()
+    >>>
+    # Apply the OpenMP Taskloop transformation to *every* loop
+    # in the schedule.
+    # FIXME: This ignores loop dependencies
```

Comment:
We usually don't have FIXMEs in master.
If this going to be fixed, can you convert it to a TODO and create an issue if it doesn't already exist.
If not and this interface is supposed to stay this way and a future meta-transformation contain the loop dependency logic just convert it to a regular comment.

Reply:
Changing this to a regular comment, with a TODO for the taskwait directive coming with #1368